.pre-commit-config.yaml

```diff
@@ -28,7 +28,7 @@ repos:
         exclude: |
           (?x)(
             ^integration-tests/sceptre-project/config/|
-            ^integration-tests/sceptre-project/templates/jinja/|
+            ^integration-tests/sceptre-project/templates/|
```

Comment:
These changes are necessary from this PR https://github.com/Sceptre/sceptre/pull/1470 I merged that branch in here so that the build could be green.